gflownet.py

```diff
@@ -823,8 +937,10 @@ def flowmatch_loss(self, it, batch, loginf=1000):
                 (1 - done).sum() + 1e-20
             )
 
-        if self.tau > 0:
-            for a, b in zip(self.model.parameters(), self.target.parameters()):
```

Comment:
Since there seemed to be an operation just on the `model.parameters()`, I initialized `self.target=None` when `forward_policy=="uniform`.